config/Darwin_Arm

```diff
@@ -0,0 +1,68 @@
+/*
+ *	Description:	This file contains the configuration for a
+ *                      gfortran/gcc build on a *64 bit* ARM (Apple Silicon) Mac system.
+ *
+ *                      You may need to modify it to change or include
+ *                      options. This version was tested with GCC-12.2
+ *                      built with Homebrew and XCode 13.4.1 on OS 12.6 (Monterey)
+ *
+ *                       Note: if the build has trouble finding the
+ *                       "gfortran" library, then you may need to modify
+ *                       the "CtoFLibraries" line below and include a "-L"
+ *                       path to help it. For example:
+ *
+ *     #define CtoFLibraries  -L/usr/local/lib -lgfortran -lquadmath
+ *     or
+ *     #define CtoFLibraries  /usr/local/lib/libgfortran.a /usr/local/lib/libquadmath.a
+ *
+ *                       You'll need to change "/usr/local/lib" to
+ *                       whatever directory contains "libgfortran.so".
+ *
+ *			 The references to the "User" macros below is for building the
+ *                       ncargcc, ncargf90, nhlcc, etc, scripts.  We don't want the
+ *                       development stuff (like the CtoFLibraries) jammed into those
+ *                       scripts, like "/usr/local/lib/libcairo.a"; instead, we want
+ *                       clean paths like "-lcairo".
+ *  #define CtoFLibraries    -L/Users/Shared/homebrew/Cellar/gcc/12.2.0/lib/gcc/current  -lgfortran -lquadmath
+ * #define CtoFLibrariesUser -L/Users/Shared/homebrew/Cellar/gcc/12.2.0/lib/gcc/current -lgfortran -lquadmath
+ */
+#define DEBUG 1
+#define HdfDefines  -DDARWIN
+#define StdDefines  -DSYSV -D_POSIX_SOURCE -D_XOPEN_SOURCE -DByteSwapped -D__UNIXOS2__ -D_DARWIN_C_SOURCE
+#define ByteSwapped
+#define Cstatic 
+#define Cdynamic 
+#define CppCommand '/usr/bin/cpp -traditional'
+#define CCompiler   gcc-12
+#define CxxCompiler   g\+\+-12
+#define FCompiler   gfortran-12
+#define CcOptions      -ansi -fPIC -Wall -std=c99 -Wno-error=implicit-function-declaration -O
```

Comment:
Don't disable the implicit function declaration error. This condition was made an error for a reason. It is important to fix it properly by declaring all functions before you use them.

Reply:
I believe it used to be OK to have implicit declarations, but more recent compilers have started treating these as errors. I'm not the one to fix all those, and if I recall there are a lot of them.